include/ofi_hmem.h

```diff
@@ -170,6 +170,22 @@ int neuron_hmem_cleanup(void);
 void *neuron_alloc(void **handle, size_t size);
 void neuron_free(void **handle);
 
+int synapseai_init(void);
+int synapseai_cleanup(void);
+int synapseai_copy_to_hmem(uint64_t device, void *dest, const void *src,
+						size_t size);
```

Comment:
The alignment is weird. One of the common conventions here is to align with the first argument. i.e. `size_t` aligns with `uint64_t`. Same for other places.

Reply:
Thank you! Fixed it the newer revision.